content/integrations/awsdirectconnect.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Datadog-AWS Direct Connect Integration
+integration_title: AWS Direct Connect
+kind: integration
+git_integration_title: amazon_directconnect
+doclevel: basic
+---
+
+## Overview
+
+AWS Direct Connect makes it easy to establish a dedicated network connection from your premises to AWS.
```

Comment:
Our integration overviews aren't in the habit of explaining what the integrated-to thing is. If a user that lands here doesn't know what AWS Direct Connect is, it's a good bet he's not going to need this integration.